package.json

```diff
@@ -59,6 +59,7 @@
   },
   "dependencies": {
     "date-fns": "2.30.0",
+    "pixi-viewport": "^5.0.2",
     "pixi.js": "^7.3.1"
```

Comment:
Both of these probably need to be pinned

Reply:
though honestly don't need to be during development, now that i think about it

Reply:
Agreed, will pin in the morning

Reply:
Updated